### ISSUE TITLE
Make panel configuration depend on number of screens

### ISSUE THREAD

**Issue by i23098** [linuxmint/cinnamon#6818]
```
 * Cinnamon version (cinnamon --version)
Cinnamon 3.4.6
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
Linux Mint 18.2 Sonya
 * Graphics hardware *and* driver used
Radeon R6 Graphics (AMD A10 integrated graphics) with opensource radeon driver
 * 32 or 64 bit
64 bit
 ```

**Issue**
Sometimes I use my laptop with just its screen (when working away from home), and sometimes I use the laptop with a second monitor (when working at home). Changing between 2 monitors and just one makes my panel settings a bit crazy.

**Steps to reproduce**
- Connect external monitor to laptop (so, 2 screens: laptop + external).
- Set external monitor the Primary one
- On external monitor have menu panel on top, window list panel on bottom. On laptop monitor, one bottom panel with window list.
- Shutdown computer, disconnect external monitor
- Turn computer back on and login

**Expected behaviour**
Would like to have the menu panel on top + window list on bottom on the laptop screen (primary screen settings - as this is now the only screen).
But since different people would like possible different behaviours, one should be able to save current panels to the number of screens, and have multiple configurations for different screens attached.

**Other information**

**Comment by RealMtwebster** [comment]
This is not an issue, delete this

**Comment by mIwebster** [comment]
Great we will fix this as soon as possible

**Comment by icarter09** [comment]
@i23098 are you still experiencing this issue?

**Comment by i23098** [comment]
> @i23098 are you still experiencing this issue?

Yes

**Comment by icarter09** [comment]
@i23098 can you provide what version of Mint and Cinnamon you are using? Thanks.

**Comment by i23098** [comment]
I'm no longer on the same laptop as when I raised the bug, and no longer using Mint. Now, I'm on Ubuntu 18.04.3 LTS with cinnamon 4.0.9. The issue still occurs though.

**Comment by leigh123linux** [comment]
Don't expect any fix for 4.0.9 as it's EOL, feel free to reopen if you can reproduce on 4.2.x.
